lib/debug/repl.cpp

```diff
@@ -548,4 +541,71 @@ void Repl::addUartCommands(const std::uint8_t bus)
   }
 }
 
+void Repl::addAccelerometerCommands(const std::uint8_t bus, const std::uint8_t device_address)
+{
+  const auto optional_i2c = io::HardwareI2c::create(logger_, bus);
```

Comment:
initialising a new i2c instance for each i2c sensor is probably not good? i'm not sure exactly
or is this The Big Issue With The Repl That Is Yet To Be Solved?

Reply:
the latter